dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/PersistenceReport.java

```diff
@@ -45,40 +45,30 @@
  *
  * @author Morten Olav Hansen <mortenoh@gmail.com>
  */
-@Data
-@Builder
+@Getter
 @NoArgsConstructor
 @AllArgsConstructor
-public class TrackerBundleReport
+public class PersistenceReport
 {
-    @JsonProperty
-    @Builder.Default
-    private TrackerStatus status = TrackerStatus.OK;
+    public static PersistenceReport emptyReport()
```

Comment:
Is it a good idea to have/useful to have an empty report?

Reply:
Maybe not, I will however think about it in next PR(s) as we need to have the case when this report is empty/null that is when there are validation errors and nothing was persisted